### PR TITLE
temporarily remove unit-threaded until failure can be investigated

### PR DESCRIPTION
**Pull request by JinShil**
This is preventing some PRs from being merged.

### CODE REVIEW

**Comment by wilzbach** [comment]
CC @atilaneves

**Comment by wilzbach** [comment]
Example: https://ci.dlang.io/blue/organizations/jenkins/dlang-org%2Fdmd/detail/PR-7613/1/pipeline

**Comment by wilzbach** [comment]
This is quite crude. Would you mind doing it in a similar way like Vibe.d does? I mean the ` || echo failed` - the advantage is that one still sees the error message on the CI.

And now that you have a PR number, how about referencing it in the sources?

(I currently on the go - otherwise I would have made those tiny changes myself)

**Comment by JinShil** [comment]
Sorry @wilzbach, but I really don't understand.  If I follow the vibe.d example, I think I can remove just the failing tests.  I used the jenkins output to identify the failing tests and added a shell script just like vibe.d to remove the failures.  I hope that's what you mean.  If not, please elaborate.  I'm sorry, but I really don't understand how the logic in this .groovy file works.

File: vars/runPipeline.groovy [hunk 1]
```diff
@@ -207,6 +207,16 @@ def testDownstreamProject (name) {
...
                     break;
 
+                // FIXME
+                // temporarily remove failing unit-threaded tests until they can be investigated
+                case 'atilaneves/unit-threaded':
+                    sh 'rm -rf tests/module_with_attrs.d'
+                    sh 'rm -rf tests/module_with_setup.d'
+                    sh 'rm -rf tests/module_with_tests.d'
+                    sh 'rm -rf tests/parametrized.d'
+                    sh 'rm -rf tests/tags.d'
```
Comment (wilzbach) [inline vars/runPipeline.groovy]: You remove file that don't exist (https://github.com/atilaneves/unit-threaded/tree/master/tests) and don't even execute the testsuite here.

**Comment by atilaneves** [comment]
This has already been fixed in unit-threaded.

**Comment by wilzbach** [comment]
@atilaneves: the Jenkins project tester checks out the latest tag by default.
If that has been fixed, it's great. Would you be so kind to open a PR and remove the workaround. I have a feeling that you are the best person for this as it will be easy for you to take actions in case something still doesn't work.

**Comment by atilaneves** [comment]
@wilzbach On it.

**Pull request merged**